src/abstractdataframe/iteration.jl

```diff
@@ -19,9 +19,9 @@ which is a view that acts like a one-row DataFrame.
 eachrow(df::AbstractDataFrame) = DFRowIterator(df)
 
 Base.start(itr::DFRowIterator) = 1
-Base.done(itr::DFRowIterator, i::Int) = i > size(itr.df, 1)
+Base.done(itr::DFRowIterator, i::Int) = i > length(itr)
```

Comment:
Please keep this as-is, there have been discussions about deprecating `length` altogether.